<launch>
    <node pkg="lidar_aggregation" type="lidar_aggregation_node" name="end_time_lidar_aggregator" output="screen">
        <rosparam command="load" file="$(find lidar_aggregation)/config/end_time_aggregator_params.yaml" ns="/lidar_aggregation/time_publisher" />
    </node>
</launch>
